server.js

```diff
@@ -155,7 +155,9 @@ if (AppConstants.FXA_ENABLED) {
 app.use("/scan", ScanRoutes);
 app.use("/ses", SesRoutes);
 app.use("/user", UserRoutes);
-app.use("/email-l10n", EmailL10nRoutes);
+if (app.get("env") === "dev") {
```

Comment:
On heroku we have:
```
NODE_ENV:                            heroku
```
So, this should be `if (app.get("env") === "heroku") {`

In fact, we could move this up into the existing `heroku` check at https://github.com/mozilla/blurts-server/blob/master/server.js#L73-L82

Reply:
@groovecoder updated. I wasn't able to view locally with just ==="heroku" but maybe I am goofing something?

Reply:
Would you need to run `$ NODE_ENV=heroku npm start`?

Reply:
Works! But I think anyone trying to view the email-l10n routes locally will have to have a self-signed cert for their localhost or disable the https redirect.